src/utils/InvertedMirror.js

```diff
@@ -29,23 +29,23 @@ export default class InvertedMirror extends PureComponent {
         <div className='row'>
           <div className='col-md-6'>
             <Background
-              color='light'
-              className='mc-card mc-invert'
+              color='dark'
+              className='mc-card'
             >
               <h5 className='mc-text-h5 mc-mb-7 mc-text--center'>
-                Inverted
+                Dark Theme (default)
               </h5>
               {children}
             </Background>
           </div>
 
           <div className='col-md-6'>
             <Background
-              color='dark'
-              className='mc-card'
+              color='light'
+              className='mc-card mc-invert'
```

Comment:
should prolly use `.mc-theme-light` here

```suggestion
              className='mc-card mc-theme-light'
```